### ISSUE TITLE
mAP too low when training yolov4 tiny custom model

### ISSUE THREAD

**Issue by andreluizeng** [AlexeyAB/darknet#8850]
Hi there, I have been trying to train my custom yolov4 tiny model but it seems like there are some issues when calculating the mAP, also the avg loss is too high. If I get the same config and image dataset and use the 795474f0ed022fd73af4e70cac79751f672d8af9 commit, it works perfectly. Is there any known issue ?

cheers,
Andre

**Comment by cenit** [comment]
no open issues, did you find the same behaviour with any open dataset so I can try to reproduce your issue?

**Comment by andreluizeng** [comment]
> no open issues, did you find the same behaviour with any open dataset so I can try to reproduce your issue?

Unfortunately not, but it is a dataset that I used in a project already and worked pretty good, mAP at 99.8%. i am training it again using the commit i mentioned and it is working good so far.

**Comment by cenit** [comment]
the fact is that it should be the same with the most recent one, given the same parameters of course...

**Comment by andreluizeng** [comment]
> the fact is that it should be the same with the most recent one, given the same parameters of course...

agreed, there is something wrong. I also tried the Yolov4 tag and it also has the same problem, the only one I can get working is the commit I mentinoned before.
